recipes/openpmd-validator/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "openpmd-validator" %}
+{% set version = "1.1.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/openPMD/openPMD-validator/archive/{{ version }}.tar.gz
+  sha256: a15153c95350e713866ff44a307548f575bd4a7833b082d65fe2d5d0551d48f8
+  patches:
+    # remove checked-in meta.yaml file
+    - rm-meta.patch
+
+build:
+  number: 0
```

Comment:
Ooops, forgot how to express `noarch`.
Here is the fix: https://github.com/conda-forge/openpmd-validator-feedstock/pull/2